cypress/integration/LeftNavigationClickEvent/beforeEach.js

```diff
@@ -0,0 +1,89 @@
+beforeEach(() => {
+    // Create an image proxy to handle intercepting
+    // analytics calls.
+
+    cy.on('window:before:load', (win) => {
+    
+       cy.AnalyticsStorage = [];
+      const NativeImage = Image;
+  
+      class AnalyticsInterceptingImage {
+        constructor(w, h) {
+          const nativeImage = new NativeImage(w, h);
+          const handler = {
+            set: function (obj, prop, value) {
+              if (prop === 'src') {
+                // TODO: Filter out non Adobe URLs
```

Comment:
These todos need to be to-did. :)

Reply:
As a reminder:
1. Adobe URLs are *.2o7.net or metrics.cancer.gov.
2. A nice analytics object should be the URL converted into nice human readable properties. (i.e. NOT a query string)
    * The list of params is at https://docs.adobe.com/content/help/en/analytics/implementation/validate/query-parameters.html
       * THE COLUMN TITLED "Analytics implementation variable" should be used for naming the property.
    * The biggest reason for this is that people will use the s object names for things, such as prop10, instead of c10. In fact no one other than you knows what c10 is. (Well, I guess I do now)

Reply:
1)Filtered out non-adobe by checking if value contains tracking server
2)Beacon now contains Analytics implementation variable parameter name when provided